wandb/cli/cli.py

```diff
@@ -1191,6 +1192,13 @@ def launch_sweep(
     help="""Path to JSON file (must end in '.json') or JSON string which will be passed
     as a launch config. Dictation how the launched run will be configured.""",
 )
+@click.option(
+    "--fields",
+    "-f",
+    default=None,
+    multiple=True,
+    help="""Fields for queues with allowlisting enabled, as key-value pairs e.g. `--fields key1=value1 --fields key2=value2`""",
+)
```

Comment:
We should error if this is not but `--queue` is not

Reply:
I'm also not sure about the name `--fields`. We call these `launch time overrides` in the app. I'm also worried that this will be confused with the run config overrides that we perform and should probably have a cli for as well :/

Reply:
Helm uses `--set` on the command line, which I quite like actually. But I'm wondering if we should have a way to distinguish these from run config overrides. `--set-template` and `--set-config` perhaps?

Reply:
> `--set-template` and `--set-config` perhaps?

Are you saying we do this?:

`--fields` -> `--set-template`
`--config` -> `--set-config`

Given how differently these flags are used, I think this kind of parallelism in the flag names would be misleading.

I agree that `--fields` is not good. At the very least, the singular `--field` would make more sense, since each flag sets one field.

I keep changing my mind about `--set` but I think I like it. In the scenario where a team has set up the queue with allowlisting, it makes sense to use the simple verb "set" to set the parameters for this particular launched job.

Reply:
`--template-variable` is nice and descriptive but feels way too long. But maybe that doesn't even matter since there'll be a one-character alternative